funcs.sh

```diff
@@ -68,12 +68,12 @@ function composer_build {
 	composer run-script cloud-build
 }
 
-function set_node_version {
+function node_install {
 	. /root/.nvm/nvm.sh --no-use
 
 	if [[ -f ".nvmrc" ]]; then
-		echo "nvm use"
-		nvm use
+		echo "nvm install"
+		nvm install
 	else
 		echo "No .nvmrc found; Defaulting to Node 16"
 		nvm use 16
```

Comment:
THIS IS A POC - BUT

Would be best to install

Reply:
When we install NVM in Dockerfile, it is configured to install NodeJS16 as the "default". So 16 is already installed and `nvm use 16` is there just to ensure that 16 is selected to be used.